doc/stdlib/dates.rst

```diff
@@ -136,6 +136,11 @@ alternatively, you could call ``using Dates`` to bring all exported functions in
    Returns a DateTime corresponding to the user's system
    time as UTC/GMT.
 
+.. function:: eps(::DateTime) -> Millisecond
+              eps(::Date) -> Day
+
+   Returns ``Millisecond(1)`` for ``DateTime`` values and ``Day(1)`` for ``Date`` values.
+
```

Comment:
1. Does this doc get picked up when I wonder what `eps` is, and type `?eps` in the REPL. CC @one-more-minute 
2. The page layout seems to imply `Dates.eps`. It seems to me like that doesn't work now, but that it would work if you `import Base.eps`. I haven't tested though.